ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocolConfiguration.java

```diff
@@ -24,7 +24,7 @@
 public class EthProtocolConfiguration {
 
   public static final int DEFAULT_MAX_MESSAGE_SIZE = 10 * ByteUnits.MEGABYTE;
-  public static final int DEFAULT_MAX_GET_BLOCK_HEADERS = 192;
+  public static final int DEFAULT_MAX_GET_BLOCK_HEADERS = 512;
```

Comment:
why we need to increase that ?

Reply:
geth requests headers in blocks of 512.  their rationale is here: https://discord.com/channels/595666850260713488/1014177210560696330/1171097165473521754

If geth cannot get a full set of 512 headers, it will disconnect the peer it is trying to sync from